packages/embed/src/utils/build-iframe-src.ts

```diff
@@ -20,17 +21,21 @@ const typesToEmbed: Record<EmbedType, string> = {
   'side-tab': 'popup-side-panel',
 }
 
-const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlOptions): Record<string, any> => ({
-  'typeform-embed-id': embedId,
-  'typeform-embed': typesToEmbed[type],
-  'typeform-source': options.source,
-  'typeform-medium': options.medium,
-  'typeform-medium-version': options.mediumVersion,
-  'embed-hide-footer': options.hideFooter ? 'true' : undefined,
-  'embed-hide-headers': options.hideHeaders ? 'true' : undefined,
-  'embed-opacity': options.opacity,
-  'disable-tracking': options.disableTracking ? 'true' : undefined,
-})
+const mapOptionsToQueryParams = (type: EmbedType, embedId: string, options: UrlOptions): Record<string, any> => {
+  const transitiveParams = getTransitiveSearchParams(options.transitiveSearchParams as string[])
```

Comment:
Is the cast `as string[]` necessary? I see it is defined correctly as `transitiveSearchParams?: string[]` and the method accepts same type as parameter.